pkg/services/postgresqldb/types_all_in_one.go

```diff
@@ -0,0 +1,47 @@
+package postgresqldb
+
+import "github.com/Azure/open-service-broker-azure/pkg/service"
+
+// AllInOneProvisioningParameters encapsulates both dbms and database
+// PostgreSQL-specific provisioning options
+type AllInOneProvisioningParameters struct {
+	SSLEnforcement  string   `json:"sslEnforcement"`
+	FirewallIPStart string   `json:"firewallStartIPAddress"`
+	FirewallIPEnd   string   `json:"firewallEndIPAddress"`
+	Extensions      []string `json:"extensions"`
+}
+
+type allInOnePostgresqlInstanceDetails struct {
+	dbmsOnlyPostgresqlInstanceDetails
```

Comment:
Any reason to do this here and _not_ use the same trick for the provisioning params above? (There may be. I'm asking.)

Reply:
No, I think I decided to unroll it for readability and forgot to clean this one up. I can go back to the embedded version if you'd like!

Reply:
Only if it doesn't complicate things. Sometimes that does. Dealer's choice.